vignettes/datatable-sd-usage.Rmd

```diff
@@ -73,7 +73,15 @@ The first way to impact what `.SD` is is to limit the _columns_ contained in `.S
 Pitching[ , .SD, .SDcols = c('W', 'L', 'G')]
```

Comment:
Update the Stack Overflow post ???

https://stackoverflow.com/questions/8508482/what-does-sd-stand-for-in-data-table-in-r/47406952#47406952